pkgs/stdenv/darwin/default.nix

```
@@ -651,7 +651,7 @@ rec {
         darwin = super.darwin.overrideScope (_: _: {
           inherit (darwin) dyld ICU Libsystem Csu libiconv rewrite-tbd;
         } // lib.optionalAttrs (super.stdenv.targetPlatform == localSystem) {
-          inherit (darwin) binutils binutils-unwrapped cctools;
+          inherit (darwin) binutils binutils-unwrapped cctools cctools-port;
```

```suggestion
          inherit (darwin) binutils binutils-unwrapped cctools-port;
```

Looks like we can drop cctools because it's just `cctools-port` anyway.

---

I think we’d want the override to make sure darwin.cctools is the same as what’s used in the stdenv. Dropping cctools by itself should be okay though.

---

Reasons in favor of keeping the override:

- One fewer build of the same stuff

Reasons in favor of dropping the override:

- Reduces complexity of the `stdenv/default.nix`

---

I pushed a commit that drops `cctools` and left `cctools-port`.

```fish
$ nix-build --pure --no-out-link -A darwin.cctools -A darwin.cctools-port
/nix/store/3p2xdd5xmrc0qyi10lf95nbydcyprqy9-cctools-port-973.0.1
/nix/store/3p2xdd5xmrc0qyi10lf95nbydcyprqy9-cctools-port-973.0.1
```